tasks/main.yml

```diff
@@ -488,14 +500,29 @@
 
     - name: Add the fixed mssql profile to the list of Tuned profiles
       command: >-
-        tuned-adm profile {{ __mssql_tuned_active_profiles.stdout |
-        regex_replace('^Current active profile: ', '') }} mssql
+        tuned-adm profile {{ __mssql_tuned_active_profiles.content }} mssql
```

Comment:
```suggestion
        tuned-adm profile {{ __mssql_tuned_active_profiles.content | b64decode }} mssql
```

Reply:
I centralized that into a new fact, as there are too many places which use this value.